.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Please, add `test.md` on the .gitngnore file. 🐦

Reply:
how?